consent/consent-page/examples/docker-compose-totp-styles.yaml

```diff
@@ -0,0 +1,230 @@
+      - LOGIN_AUTHORIZATION_SERVER_URL=${ACP_URL}
+      - LOGIN_AUTHORIZATION_SERVER_ID=bank-admins
+      - LOGIN_TENANT_ID=${TENANT}
+      - LOGIN_CLIENT_ID=bv2f80aed9c9is80f2d0
+      - INTROSPECT_CLIENT_ID=bv2fddlpfc67lmeti32g
+      - INTROSPECT_CLIENT_SECRET=RkAIOVRlP6ZLhDvYixT1wBb8DG3bVkXAouoIrX7cwzA
+      - INTROSPECT_ISSUER_URL=${ACP_INTERNAL_URL}/${TENANT}/bank-admins
+
+  bank:
+    image: cloudentity/openbanking-quickstart-bank:${VERSION}
+    container_name: bank
+    restart: always
+    ports:
+      - "8070:8070"
+    volumes:
+      - ./data/ca.pem:/ca.pem
+      - ./data/bank_cert.pem:/bank_cert.pem
+      - ./data/bank_key.pem:/bank_key.pem
+      - ./mount/bank:/app/data
+    environment:
+      - CLIENT_ID=bukj5p6k7qdmm5ppbi4g
+      - ISSUER_URL=${ACP_INTERNAL_URL}/${TENANT}/openbanking
+      - CERT_FILE=/bank_cert.pem
+      - KEY_FILE=/bank_key.pem
+      - ROOT_CA=/ca.pem
+
+  acp:
+    container_name: acp
+    restart: always
+    image: ${ACP_DOCKER_IMAGE}:${ACP_VERSION}
+    ports:
+      - 8443:8443
+    volumes:
+      - ./data/acp_cert.pem:/acp_cert.pem
+      - ./data/acp_key.pem:/acp_key.pem
+    environment:
+      - FEATURES_SWAGGER_UI=true
+      - FEATURES_INTEGRATION_ENDPOINTS=true
+      - FEATURES_SYSTEM_OPENBANKING_CONSENTS_MANAGEMENT=true
+      - FEATURES_OPENBANKING_DOMESTIC_PAYMENT_CONSENTS=true
+      - FEATURES_SYSTEM_CLIENTS_MANAGEMENT=true
+      - SERVER_URL=${ACP_URL}
+    command:
+      - server
+      - start
+      - --sql.url
+      - postgres://root@crdb:26257/defaultdb?sslmode=disable
+      - --hazelcast.address
+      - hazelcast:5701
+      - --secret.key
+      - KNEcLGdDqpwrXDubqPgDSUkMMsLPXaHh
+      - --server.cert.path
+      - /acp_cert.pem
+      - --server.key.path
+      - /acp_key.pem
+      - --demo
+      - --create-default-tenant
+
+  crdb:
+    container_name: crdb
+    image: cockroachdb/cockroach:v20.2.6
+    restart: always
+    ports:
+      - 26257:26257
+      - 8081:8080
+    command: start-single-node --insecure
+    volumes:
+      - crdb:/cockroach/cockroach-data"
+
+  hazelcast:
+    container_name: hazelcast
+    image: hazelcast/hazelcast:3.12
+    restart: on-failure
+    ports:
+      - 5701:5701
+
+  configuration:
+    container_name: configuration
+    image: cloudentity/openbanking-quickstart-configuration:latest
+    restart: on-failure
+    depends_on:
+      - acp
+    volumes:
+      - ./data/tpp_cert.pem:/certs/tpp_cert.pem
+      - ./data/ca.pem:/certs/ca.pem
+      - ./data/variables.yaml:/variables.yaml
+      - ./data/imports:/app/imports
+    command:
+      - /app/main
+      - --tenant-url
+      - ${CONFIGURATION_TENANT_URL}
+      - --tenant
+      - ${CONFIGURATION_TENANT}
+      - --client-id
+      - ${CONFIGURATION_CLIENT_ID}
+      - --client-secret
+      - ${CONFIGURATION_CLIENT_SECRET}
+      - --templates-dirs
+      - /app/imports
+      - --variables-file
+      - /variables.yaml
+      - --verbose
+
+volumes:
+  crdb:
```

Comment:
Is this necessary to copy the entire docker-compose? Would it be possible to override just the service that we are overriding?